### DISCUSSION TITLE
请求皇家守卫军

### DISCUSSION THREAD

**Discussion by 972933728** [Gamuxorg/bbs#232] [投递和请求游戏]
皇家守卫军第一代就行了，前线和起源可以用wine就行。但是第一代如果wine很卡。

**Comment by shouhuanxiaoji** [comment]
您好，由于正在全力编写新版网站，可能不能及时回复您的请求，抱歉

**Comment by liushengqi000** [comment]
KINGDOM RUSH 第一代是flash的吧？后来似乎也是flash封装成exe的。

**Comment by 972933728** [comment]
> KINGDOM RUSH 第一代是flash的吧？后来似乎也是flash封装成exe的。

不是。第一代有pc版的，不是flash的。

**Comment by shouhuanxiaoji** [comment]
已处理
